ruleset/rules/0920-oracledb_rules.xml

```diff
@@ -0,0 +1,36 @@
+<!--
+  - OracleDB rules
+  - Created by Wazuh, Inc.
+  - Copyright (C) 2015-2021, Wazuh Inc.
+  - This program is a free software; you can redistribute it and/or modify it under the terms of GPLv2.
+-->
+
+<group name="oracle,oracledb,">
+
+    <!-- RULE ID RANGE: 89100 - 89199 -->
+
+    <rule id="89100" level="0">
+        <decoded_as>oracledb_log</decoded_as>
+        <description>Oracle DB</description>
+    </rule>
+
+    <!--
+      Alert example:
+      {"timestamp":"2021-07-14T11:21:33.363-0700","rule":{"level":3,"description":"OracleDB transaction","id":"89101","firedtimes":1,"mail":false,"groups":["oracle","oracledb"]},"agent":{"id":"000","name":"localhost.localdomain"},"manager":{"name":"localhost.localdomain"},"id":"1626286893.177547","full_log":"Jul 14 11:21:33 localhost.localdomain oracledb: \nFri Jul 09 11:11:00 2021 +05:32\nLENGTH : '563'\nACTION :[412] 'select /*+  no_parallel(t) no_parallel_index(t) dbms_stats cursor_sharing_exact use_weak_name_resl dynamic_sampling(0) no_monitoring xmlindex_sel_idx_tbl no_substrb_pad  */ substrb(dump(\"INSTANCE_NUMBER\",16,0,64),1,240) val, \n                      rowidtochar(rowid) rwid from \"SYS\".\"WWW$_SEC_THAT\" t where rowid in (chartorowid('AAAAO0AADAAASALAAA'),chartorowid('AAAAO0AADAAAAAJAAA')) order by \"INSTANCE_NUMBER\"'\nDATABASE USER:[3] 'SYS'\nPRIVILEGE :[4] 'NONE'\nCLIENT USER:[0] ''\nCLIENT TERMINAL:[7] 'UNKNOWN'\nSTATUS:[1] '0'\nDBID:[10] '1120301032'","predecoder":{"program_name":"oracledb","timestamp":"Jul 14 11:21:33","hostname":"localhost.localdomain"},"decoder":{"name":"oracledb_log"},"data":{"action":"select /*+  no_parallel(t) no_parallel_index(t) dbms_stats cursor_sharing_exact use_weak_name_resl dynamic_sampling(0) no_monitoring xmlindex_sel_idx_tbl no_substrb_pad  */ substrb(dump(\"INSTANCE_NUMBER\",16,0,64),1,240) val, \n                      rowidtochar(rowid) rwid from \"SYS\".\"WWW$_SEC_THAT\" t where rowid in (chartorowid('AAAAO0AADAAASALAAA'),chartorowid('AAAAO0AADAAAAAJAAA')) order by \"INSTANCE_NUMBER\"","status":"0","length":"563","database_user":"SYS","privilege_level":"NONE","client_terminal":"UNKNOWN","database_id":"1120301032"},"location":"/tmp/oracle/CDBTST000_j000_11111_20212020231542600818143795.aud"}    
+    -->
+    <rule id="89101" level="3">
```

Comment:
As a suggestion I would merge rules 81900 and 81901 unless another reason for rule granularity shows up

Reply:
Done